session-replay/Tests/DatadogSessionReplayTests/Recorder/ViewTreeSnapshotProducer/Utilties/UIImageView+RecordingTests.swift

```diff
@@ -0,0 +1,173 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+import UIKit
+@testable import DatadogSessionReplay
+
+/// Make sure to run on iPhone 11
```

Comment:
This won't be sustainable - we must keep the freedom of changing CI stacks / simulators without requiring updates in tests. Also, tests should pass no matter of the environment we use for the sake of simulator x OS version randomization we do in [bitrise-nightly-unit-tests.yml](https://github.com/DataDog/dd-sdk-ios/blob/develop/tools/nightly-unit-tests/bitrise-nightly-unit-tests.yml) (that will be eventually enabled for SR).
```suggestion
```